docker/dev/README.md

```diff
@@ -0,0 +1,69 @@
+# Environment: Development
+
+This environment provides a mock backend for local development and E2E testing.  
+The mock backend simulates the LinID API metadata endpoints.
+
+## Available Endpoints
+
+| Method | Path                         | Description                       |
+| ------ | ---------------------------- | --------------------------------- |
+| GET    | `/health`                    | Health check                      |
+| GET    | `/metadata/routes`           | Returns all route configurations  |
+| GET    | `/metadata/entities`         | Returns all entity configurations |
+| GET    | `/metadata/entities/:entity` | Returns a specific entity config  |
+
+## Quick Start
```

Comment:
quickstart should be more quick
only:
docker build -t linid-mock-backend docker/mock-backend/
docker container prune -f
docker compose -f docker/dev/docker-compose.yml up

remove other